### PR TITLE
Fixes #58 - Add txAdmin volume information

### LINKED ISSUE #58: README needs txAdmin volume information
As the title says, I had to reverse engineer a few things to find where the `txAdmin` stores it's files, as restarting the docker container kept resetting the values.

Submitting a PR.

### PR DESCRIPTION
**Pull request by CodexHere**
Fixes #58

Added quick notation of how to volume mount for txAdmin data.

### CODE REVIEW

**Review by frebib — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -46,6 +46,21 @@ This image has two tags - a `latest` tag (the default), based on the most recent
 
 The web UI can be enabled by not passing any `+exec` config to the FXServer binary. This can be achieved by setting the `NO_DEFAULT_CONFIG` environment variable (see below).
 
+`txAdmin` stores it's configuration and database data in `/txAdmin`, so a volume can be set up to persist this data:
```
Comment (Adam-Ant) [inline README.md:49]: The path here doesn't match the compose file below - it should be `/txData` I think?

File: README.md [hunk 2]
```diff
@@ -46,6 +46,21 @@ This image has two tags - a `latest` tag (the default), based on the most recent
...
+
+```sh
+docker run -d \
+  --name FiveM \
+  --restart=on-failure \
+  -e LICENSE_KEY=<your-license-here> \
+  -p 30120:30120 \
+  -p 30120:30120/udp \
```
Comment (Adam-Ant) [inline README.md:57]: Port for txAdmin should be added here

File: docker-compose.yml [hunk 3]
```diff
@@ -9,6 +9,7 @@ services:
     tty: true
     volumes:
       - "/path/to/resources/folder:/config"
+      - "/path/to/txData/folder:/txData"
```
Comment (Adam-Ant) [inline docker-compose.yml:12]: As this compose file needs to cover all use cases, a comment explaining this is only for txAdmin would be good

File: docker-compose.yml [hunk 4]
```diff
@@ -9,6 +9,7 @@ services:
...
     ports:
       - "30120:30120"
       - "30120:30120/udp"
```
Comment (Adam-Ant) [inline docker-compose.yml:15]: Again - port for txAdmin & note explaining its for txAdmin

**Review by Adam-Ant — CHANGES_REQUESTED** [review]
Many thanks for putting this together - txAdmin documentation has definitely been lacking! A few small changes & additions needed then I'll be happy to merge this.

**Comment by CodexHere** [comment]
Updated with your requests

**Review by Adam-Ant — APPROVED** [review]
All looks good - many thanks for those changes and your PR!

**Pull request merged**